### ISSUE TITLE
New issue vs commenting in closed ones (contributing guide)

### ISSUE THREAD

**Issue by maelle** [tidyverse/tidyverse.org#565]
:wave: here! Is there some official guidance recommending to open new issues rather than to comment on old closed issues? I was hoping to add such a link in an answer to a comment in a closed issue. :slightly_smiling_face: (sorry if I missed something obvious!)

**Comment by juliasilge** [comment]
I don't believe we have this guidance outlined anywhere but perhaps we should at `/contribute`. 

In tidymodels, we added [this lockbot](https://github.com/tidymodels/rsample/blob/main/.github/workflows/lock.yaml) to all our repos to lock closed issues/PRs after 14 days. The messaging folks see is:

> This issue / pull request has been automatically locked. If you believe you have found a related problem, please file a new issue (with a reprex: <https://reprex.tidyverse.org>) and link to this issue.

For issues that are within the 14 days, what we do is either re-open if that's appropriate or just tell folks to open a new issue. For repos with a lot of activity, I really do think it is worth setting up the lock bots.
